client/src/modules/stock/inventories/registry.js

```diff
@@ -188,6 +191,11 @@ function StockInventoriesController(
   vm.openColumnConfigModal = openColumnConfigModal;
   vm.clearGridState = clearGridState;
 
+  // shipment in transit
+  vm.isInTransitOrPartial = status => {
+    return bhConstants.shipmentStatus.IN_TRANSIT === status || bhConstants.shipmentStatus.PARTIAL === status;
```

Comment:
A very minor stylistic thing:  Usually, the item being checked is first and the constant it is being checked against is second.  Eg:

   status === bhConstants.shipmentStatus.IN_TRANSIT

I think this reads more naturally and easier for someone reading the code to understand.

Reply:
RESOLVED 👍🏽